Makefile

```diff
@@ -136,6 +143,26 @@ docker-run:
 	@docker-compose up --build
 	@echo "++\n*****"
 
+api-unit-test:
+	@echo "++\n***** Running API unit tests\n++"
+	@yarn workspace @ehpr/api build
+	@yarn workspace @ehpr/api test
+	@echo "++\n*****"
+
+start-test-db:
+	NODE_ENV=test docker-compose -f docker-compose.test.yaml up --build -d
```

Comment:
I wanted to make it a completely separate database so it could be configured slightly differently and so it wouldn't interfere with the main app if it's already running, what do you think?

Reply:
Normally we wiped-out (tear down) db for correctness of test and isolation of test execution. In that case using separate database is good choice.